src/authentication-service.ts

```diff
@@ -167,7 +167,7 @@ export class RedHatAuthenticationService {
           try {
             await this.refreshToken(session.refreshToken, session.scope, session.id);
           } catch (e) {
-            if (e.message === REFRESH_NETWORK_FAILURE) {
+            if ((e as Error).message === REFRESH_NETWORK_FAILURE) {
```

Comment:
Would be better to declare e as Error

Reply:
That would be eslint violation, because it can be only declared as `unknown`.

Reply:
you could use `instanceof Error`